pkg/kubelet/network/dns/dns.go

```diff
@@ -237,7 +237,7 @@ func parseResolvConf(reader io.Reader) (nameservers []string, searches []string,
 			}
 		}
 		if fields[0] == "options" {
-			options = fields[1:]
+			options = append(options, fields[1:]...)
```

Comment:
I'm not sure how this fixes the problem in the description, I think I'm missed something. Can you expand on this a bit?

Reply:
> I'm not sure how this fixes the problem in the description, I think I'm missed something. Can you expand on this a bit?

I just want to reserve all `options`

Reply:
Ah I see, I missed the for loop on line 216